lib/app/sanitize_redirect.js

```diff
@@ -6,8 +6,8 @@ const _ = require('underscore')
 const parse = require('url').parse
 
 const redirectFallback = '/'
-const whitelistHosts = ['internal', 'localhost', 'artsy.net']
-const whitelistProtocols = ['http:', 'https:', null]
+const allowlistHosts = ['internal', 'localhost', 'artsy.net']
```

Comment:
Nit: allowedListHosts, allowedListProtocols

Reply:
I think it's ok as it is, but I guess it's just opinions. same way as whitelist was not whiteList, I think allowlist can be a word.

Reply:
Sure, I think I saw this word before and I'm okay with it :)

Reply:
🤩